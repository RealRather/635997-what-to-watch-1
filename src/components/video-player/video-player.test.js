import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const mock = {
  id: `22`,
  name: `Bohemian`,
  src: `img/bohemian.jpg`,
  preview: {
    mp4: `https://download.trailer-480p.mp4`,
    webm: `https://Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  genre: `Dramas`,
};

it(`Render VideoPlayer`, () => {
  const tree = renderer
    .create(<VideoPlayer
      movie={mock}
      isVideoPlaying={false}
    />,
    {
      createNodeMock: (element) => element
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
