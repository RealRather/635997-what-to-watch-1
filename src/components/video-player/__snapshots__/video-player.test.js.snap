// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer 1`] = `
<video
  height="175"
  muted={true}
  poster="img/bohemian.jpg"
  width="280"
>
  <source
    src="https://download.trailer-480p.mp4"
    type="video/mp4"
  />
  <source
    src="https://Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
    type="video/webm"
  />
  Bohemian
</video>
`;
